dts/bindings/pwm/ti,mspm0-pwm.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+#
+# SPDX-License-Identifier: LicenseRef-BnOProprietary
+
+description: |
+  Driver to enable MSPM0 PWM hardware for TIMx timers (TIMA/TIMG).
+  This driver is implementing the existing Zephyr PWM API (pwm_driver_api) and
+  supports the set_cycles and get_cycles_per_sec functions.
+  Note, supported capture channel direction for now is up to Capture Channel 2 or 4
+  (depending on the timer used) and output only.
+
+compatible: "ti,mspm0-pwm"
+
+include: [base.yaml, pinctrl-device.yaml, pwm-controller.yaml]
+
+properties:
+  ti,prescaler:
+    type: int
+    default: 0
+    description: Specifies the value used to divide the input clock frequency, allowing adjustment of the effective clock rate used by the timer.
+
+  ti,mode:
+    type: int
```

Comment:
An enum string can also be useful, but this solution is OK with me. With an enum string you'd get the enum index (i.e. [0,2]) which you could use in the driver code to choose the corresponding integer value.
But the int approach here is also OK.